xwiki-rendering-syntaxes/xwiki-rendering-syntax-xhtml/src/main/java/org/xwiki/rendering/internal/parser/xhtml/wikimodel/XWikiXHTMLWhitespaceXMLFilter.java

```diff
@@ -77,9 +77,9 @@ public void setProperty(String name, Object value) throws SAXNotRecognizedExcept
     public void endCDATA() throws SAXException
     {
         if (getContent().length() > 0 && this.containsWikiSyntax) {
-                // Make sure we clean head/trail white spaces
-                trimLeadingWhiteSpaces();
-                trimTrailingWhiteSpaces();
+            // Make sure we clean head/trail white spaces
```

Comment:
General remark not related to your commit: This comment is useless. It just explains something obvious... It should explain why it's needed to remove white spaces instead.